providers/Default/EasyAbp.Abp.PhoneNumberLogin.Provider.Default/EasyAbp/Abp/PhoneNumberLogin/Provider/Default/DefaultPhoneNumberLoginVerificationCodeSender.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using Volo.Abp.Settings;
+using Volo.Abp.Sms;
+
+namespace EasyAbp.Abp.PhoneNumberLogin.Provider.TencentCloud
+{
+    public class DefaultPhoneNumberLoginVerificationCodeSender : IPhoneNumberLoginVerificationCodeSender
+    {
+        private readonly ISmsSender _smsSender;
+
+
+        public DefaultPhoneNumberLoginVerificationCodeSender(
+            ISmsSender smsSender)
+        {
+            _smsSender = smsSender;
+        }
+
+        public virtual async Task<bool> SendAsync(string phoneNumber, string code, VerificationCodeType type, string message = null)
+        {
+            SmsMessage smsMessage = new SmsMessage(phoneNumber, code);
```

Comment:
Should use the [ABP Text Templating](https://docs.abp.io/en/abp/latest/Text-Templating) feature.